### ISSUE TITLE
Include exit code in accessible buffer for failed commands

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#173839]
Sighted users can determine if a command failed via the color of or by hovering the command decoration. That's tedious for a screen reader user, so we should include this info in the accessible buffer.

**Comment by andreamah** [comment]
Is content that is written the accessible buffer available in a screen reader?

**Comment by meganrogge** [comment]
To verify, in the normal terminal buffer, run a command that fails IE `fake command`. When you have `audioCues.terminalCommandFailed` enabled, you should hear a sound
